### PR TITLE
Convert more-info-weather to LitElement

### PR DESCRIPTION
**Pull request by iantrich**

### CODE REVIEW

File: src/dialogs/more-info/controls/more-info-weather.ts [hunk 1]
```diff
@@ -0,0 +1,274 @@
...
+      <div class="flex">
+        <iron-icon icon="hass:thermometer"></iron-icon>
+        <div class="main">
+          ${this.hass.localize("ui.card.weather.attributes.temperature")}
+        </div>
+        <div>
+          ${this.stateObj.attributes.temperature} ${this.getUnit("temperature")}
+        </div>
+      </div>
+      ${this._showValue(this.stateObj.attributes.pressure)
```
Comment (bramkragten) [inline src/dialogs/more-info/controls/more-info-weather.ts]:
```suggestion
      ${this.stateObj.attributes.pressure
```

Comment (bramkragten) [reply src/dialogs/more-info/controls/more-info-weather.ts]: Drop all the `_showValue` not needed in lit

**Review by bramkragten — APPROVED** [review]

File: src/dialogs/more-info/controls/more-info-weather.ts [hunk 2]
```diff
@@ -0,0 +1,274 @@
...
+  snowy: "hass:weather-snowy",
+  "snowy-rainy": "hass:weather-snowy-rainy",
+  sunny: "hass:weather-sunny",
+  windy: "hass:weather-windy",
+  "windy-variant": "hass:weather-windy-variant",
+};
+
+@customElement("more-info-weather")
+class MoreInfoWeather extends LitElement {
+  @property() public hass!: HomeAssistant;
```
Comment (bramkragten) [inline src/dialogs/more-info/controls/more-info-weather.ts]: Should we update on every change of `hass` I think only on language change and `config.unit_system`?

Comment (iantrich) [reply src/dialogs/more-info/controls/more-info-weather.ts]: sure

**Pull request merged**